frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/virtual-hardware-tab/virtual-hardware-tab.tsx

```diff
@@ -0,0 +1,263 @@
+  const availableCDName = getAvailableCDName(storages.map((storage) => storage.disk));
+  const withProgress = wrapWithProgress(setTabLocked);
+
+  const addButtonProps = {
+    id: 'attach-cdrom',
+    className: 'pf-m-link--align-left',
+    onClick: () =>
+      withProgress(
+        vmWizardStorageModalEnhanced({
+          initialType: DiskType.CDROM,
+          blocking: true,
+          wizardReduxID,
+          availableCDName,
+        }).result,
+      ),
+    isDisabled: isLocked || disableAddCD,
+  };
+
+  return (
+    <div className="kubevirt-create-vm-modal__virtual-hardware-tab-container">
+      <Split>
+        <SplitItem isFilled>
+          <Title headingLevel="h5" size="lg">
+            Virtual Hardware
+          </Title>
+        </SplitItem>
+      </Split>
+      <Split className="kubevirt-create-vm-modal__virtual-hardware-tab-cd-title">
+        <SplitItem isFilled>
+          <Title headingLevel="h4" size="md">
+            CD-ROMs
+          </Title>
+        </SplitItem>
+      </Split>
+      {showStorages && (
+        <>
+          <div className="kubevirt-create-vm-modal__virtual-hardware-tab-main">
+            <VMDisksTable
+              columns={[
+                {
+                  title: 'Content',
+                  sortField: 'name',
+                  transforms: [sortable],
+                },
+                {
+                  title: 'Source',
+                  sortField: 'source',
+                  transforms: [sortable],
+                },
+                {
+                  title: 'Storage Class',
+                  sortField: 'storageClass',
+                  transforms: [sortable],
+                },
+                {
+                  title: '',
+                },
+              ]}
+              columnClasses={cdTableColumnClasses}
+              data={virtualStorages}
+              customData={{ isDisabled: isLocked, withProgress, removeStorage, wizardReduxID }}
+              row={VmWizardStorageRow}
+            />
+
+            <SplitItem>
+              <div className="kubevirt-add-cd-btn">
+                <Button
+                  className="pf-m-link--align-left"
+                  variant="link"
+                  icon={<PlusCircleIcon />}
+                  {...addButtonProps}
+                >
+                  {ATTACH_CD}
+                </Button>
+                {addButtonProps.isDisabled && (
+                  <Tooltip
+                    position="bottom"
+                    content="You have reached the maximum amount of CD-ROM drives"
+                  >
+                    <OutlinedQuestionCircleIcon />
+                  </Tooltip>
+                )}
+              </div>
+            </SplitItem>
+          </div>
+        </>
+      )}
+      {!showStorages && (
+        <Bullseye className="kubevirt-create-vm-modal__virtual-hardware-tab-empty-state">
+          <Title size="sm">There are no CD-ROMs currently attached.</Title>
```

Comment:
also, wouldn't bigger margin between the title and the button look better?

Reply:
This is what it looks like with the smallest spacer above. I'm not crazy about the extra space, but could be convinced otherwise :)
<img width="522" alt="Screen Shot 2019-12-02 at 2 58 58 PM" src="https://user-images.githubusercontent.com/287880/69986669-476bd580-1514-11ea-9578-fe73bc42e244.png">

Reply:
We could add it though, because on the other hand when there is no margin above it when the button is below the table it looks too tight. 
<img width="667" alt="Screen Shot 2019-12-02 at 4 40 13 PM" src="https://user-images.githubusercontent.com/287880/69993672-ae908680-1522-11ea-9f4f-acde69579ccc.png">
can you add 
`margin-top: var(--pf-global--spacer--sm);`
 to the button

Reply:
+1

Reply:
done